pkg/test/testPackages.proj

```diff
@@ -19,6 +19,8 @@
     <TargetFrameworksToExclude Include="net463" />
     <TargetFrameworksToExclude Include="net47" />
     <TargetFrameworksToExclude Include="netcoreapp2.2" />
+    <!-- We need to enable this after we start publishing the packages for netcoreapp3.1 -->
```

Comment:
We had previously chatted that we might be able to do this now using 3.0 as a proxy.  We'd just need the "KnownFrameworkReference" to copy the 3.0 entry to 5.0 temporarily.

Reply:
I am in process of doing so. there are couple of roadblocks but i am trying of resolving them.
I pushed it to check if there was any other failing part

Reply:
If there are other blockers we can peel it off into a follow up PR, but we should make sure we understand them.